File: C/1011.c
#include <stdio.h>
 
int main() {
 
    double radius;
    double pi = 3.14159;

    scanf("%lf", &radius);
    double volume = (4.0/3) * pi * pow(radius,3);
    printf("VOLUME = %.3f\n", volume);
 
    return 0;
}

File: Javascript/1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let radius = Number(lines.shift());
let pi = 3.14159;
let volume = (4.0/3) * pi * Math.pow(radius,3);
console.log("VOLUME = " + volume.toFixed(3));
